code/modules/reagents/reagent_containers/food/snacks/meat.dm

```diff
@@ -91,3 +91,24 @@
 	name = "snow strider meat"
 	desc = "A slab of nav'twir's meat, an animal native from Adhomai."
 	icon_state = "adhomai_meat"
+
+/obj/item/weapon/reagent_containers/food/snacks/meat/mice
+	name = "mice meat"
+	icon_state = "chickenbreast"
+	desc = "You have reached the epitome of poorness: eating the station's vermin."
+
+
+/obj/item/weapon/reagent_containers/food/snacks/meat/mice/Initialize()
+	. = ..()
+	reagents.add_reagent("protein", 5)
+	reagents.add_reagent("rattoxin", 1)

```

Comment:
Giving people the bubonic plague is not better, In fact, I should think it's worse because now people can turn mice into bioweapons.